src/vs/workbench/contrib/terminal/browser/terminalMenus.ts

```diff
@@ -614,6 +678,39 @@ export function setupTerminalMenus(): void {
 					},
 					group: ContextMenuGroup.Kill,
 				}
+			},
+			{
+				id: MenuId.TerminalTabContext,
+				item: {
+					command: {
+						id: TerminalCommandId.RevealInFinder,
+						title: localize('workbench.action.terminal.revealInFinder', "Reveal Current Working Directory in Finder"),
+						precondition: TerminalContextKeys.terminalHasCwdDetectionCapability
+					},
+					group: ContextMenuGroup.Navigation,
```

Comment:
For the tab menu, just above kill aligns closest:

<img width="432" alt="Screenshot 2025-01-08 at 8 48 07 am" src="https://github.com/user-attachments/assets/4e4ec44e-0975-41fb-820e-37a0cd56a6b3" />